maintainers/maintainer-list.nix

```diff
@@ -1153,6 +1153,11 @@
     github = "danharaj";
     name = "Dan Haraj";
   };
+  danielakhterov = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add danielakhterov``.